source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRIBaseInputHandler.h

```diff
@@ -9,15 +9,21 @@
 #import <Foundation/Foundation.h>
 #import <UIKit/UIKit.h>
 
+@protocol ACRInputChangeDelegate <NSObject>
+
+-(void)inputValueChanged;
+@end
+
 @protocol ACRIBaseInputHandler
 
 @property BOOL isRequired;
 @property BOOL hasValidationProperties;
 @property BOOL hasVisibilityChanged;
 @property NSString *_Nonnull id;
+@property NSMutableSet<id<ACRInputChangeDelegate>> * _Nonnull delegateSet;
 
 - (BOOL)validate:(NSError *_Nullable *_Nullable)error;
 - (void)setFocus:(BOOL)shouldBecomeFirstResponder view:(UIView *_Nullable)view;
 - (void)getInput:(NSMutableDictionary *_Nonnull)dictionary;
-
+-(void)addObserverForValueChange:(id<ACRInputChangeDelegate>_Nullable)delegate;
```

Comment:
I think, here delegate shouldn't be nullable? Because if someone is calling addObserverForValueChange then he must provide the delegate?